src/main/com/example009/radek/v3/Calculator.java

```diff
@@ -0,0 +1,66 @@
+package main.com.example009.radek.v3;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+class Calculator {
+
+    private double result;
```

Comment:
takovy detail ... promenne k promennym a servicy k servicam, kdybys tu mel vic servic a objektu, tak by v tom byl maglajs, tak to jen preskladej ty radky a nejlepe at je mezi tim mezera

Reply:
Přesunuto

Reply:
Ty promenne nemuis mit mezi sebou prazdny radek, ale to je detial. closed